games/main.tex

```diff
@@ -0,0 +1,1123 @@
+\node (pltol) [node distance=4em,left of=zero,text width=4em, align=center] {\small channel\\deposit};
+\node (pltold) [node distance=4em,left of=zerod] {};
+\draw [dashed](rtol)--(rtold);
+\draw [dashed](ltol)--(ltold);
+\draw [dashed](prtol)--(prtold);
+\draw [dashed](pltol)--(pltold);
+\end{tikzpicture}
+\end{center}
+\caption{}
+\label{fig:paymentchannel}
+\end{figure}
+\end{center}
+
+\subsection{Zero cost entry}
+
+An additional advantage of this construct is that one can start providing service with zero ether.
+Since swap can also work only one way, if a party enters the system with zero ether (newcomer),
+but connects to peers with funds (insider), first they just provide the service (and not use it)
+in order to produce a positive balance. In the simplest case insiders pay the newcomer on chain
+but there are more efficient ways of bootstrapping.
+
+If a newcomer connects to  an insider with an existing chequebook contract,
+the insider agrees to create a chequebook contract for the newcomer
+in return for consuming newcomers services.
+ When insider's service debt reaches the amount needed by for contract creation,
+ insider just sends a transaction to create the contract with newcomer as the owner.
+Once a node has its own chequebook contract, they are able
+to consume and potentially pay out more to their creditor.
+
+In order to use the contract, however, it makes sense to have a balance on it.
+Since the owner has no ether, it makes sense
+to expect that the insider peers with funds will be the ones who deposit this starting balance.
+If peers agreed that they want to save on transaction costs, it is reasonable to create
+the contract with the required initial balance in one go. This would imply waiting out
+with contract creation until the insider's debt reaches the \emph{cost of bootstrapping} which is
+the sum of (1) the cost of contract creation, (2) the required starting balance, and (3) some
+extra service fee to incentive insiders to lend.
+
+With the global deposit included, this cost of bootstrapping
+can be substantially higher than nodes' loss tolerance.
+If the contract is deployed to the blockchain in advance, with ownership granted
+to the newcomer, the newcomer can just move with their new topped-up wallet
+to other peers leaving the lender node with loss.
+If it is to be deployed only after newcomer provided service totalling
+the cost, then the insider node will disappear after essentially leeching on
+the newcomer.
+
+Luckily, there is a safe protocol to tackle
+bootstrapping: the newcomer issues a cheque to the insider in advance with an amount
+corresponding to the bootstrapping cost. Insider creates a contract with herself as owner
+and a channel deposit also dedicated to herself in the amount
+on the cheque. At this point if the two peers never meet, the newcomer suffered no loss,
+the insider can just offer the contract to the next newcomer. Otherwise the service goes
```

Comment:
> the insider can just offer the contract to the next newcomer

With the assumption that there will be always newcomers ..which would like to do buisness with it.